### PR TITLE
[Notifier] Document Amazon Notifier options

### PR DESCRIPTION
**Pull request by alamirault**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | Fix #... 
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Documentation of notifier options are in their own README

This PR prepare https://github.com/symfony/symfony-docs/issues/18652 in branch 5.4

Wording is taken from https://async-aws.com/configuration.html

### CODE REVIEW

File: src/Symfony/Component/Notifier/Bridge/AmazonSns/README.md [hunk 1]
```diff
@@ -7,9 +7,15 @@ DSN example
...
 
 ```
-AMAZON_SNS_DSN=sns://ACCESS_ID:ACCESS_KEY@default?region=REGION
+AMAZON_SNS_DSN=sns://ACCESS_ID:ACCESS_KEY@default?region=REGION&profile=PROFILE
 ```
 
+where:
+ - `ACCESS_ID` is your AWS access key id
+ - `ACCESS_KEY` is your AWS access key secret
+ - `REGION` is the AWS region targeted (optional, default: `us-east-1`)
```
Comment (stof) [inline src/Symfony/Component/Notifier/Bridge/AmazonSns/README.md:16]:
```suggestion
 - `REGION` is the targeted AWS region (optional, default: `us-east-1`)
```

File: src/Symfony/Component/Notifier/Bridge/AmazonSns/README.md [hunk 2]
```diff
@@ -7,9 +7,15 @@ DSN example
...
+ - `PROFILE` is the name of your AWS configured profile (optional, default: `default`)
```
Comment (stof) [inline src/Symfony/Component/Notifier/Bridge/AmazonSns/README.md:17]: maybe this should link to the async-aws documentation to explain what this is about.

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you Antoine.

**Comment by OskarStark** [comment]
@stof I opened a PR with your comments, sorry for being too fast 🌷 

* https://github.com/symfony/symfony/pull/51226

**Pull request merged**